test/capwap_loc_provider_SUITE.erl

```diff
@@ -49,6 +49,9 @@ all() ->
      test_error].
 
 init_per_suite(Config) ->
+    % Dummy app env value to enable the calls
+    application:set_env(capwap, location_provider, #{}),
+    ct:pal("~p", [application:get_all_env(capwap)]),
```

Comment:
`Please or remove this log or add some more info before ~p`

Reply:
ok